pkg/peertls/peertls.go

```diff
@@ -142,12 +150,23 @@ func WriteChain(w io.Writer, chain ...*x509.Certificate) error {
 		return errs.New("expected at least one certificate for writing")
 	}
 
+	var extErrs []error
```

Comment:
We recently added the `utils.ErrorGroup` that is another layer on top of `utils.CombineErrors`. It seems to be a good fit here.

So you need to declare extErrs like this:
```go
var extErrs utils.ErrorGroup
```